site/public/css/polls.css

```
@@ -39,10 +39,24 @@
     cursor: pointer;
 }
 
-.response-btn {
+.option-btn {
+    vertical-align: middle;
+}
+
+.option-btn input[type="radio"]{
     vertical-align: top;
 }
 
+.student-submit[type="submit"] {
```

If this button is unique, there's no reason why we need to specify that it is of type submit.
```suggestion
.student-submit {
```